plugins/catalog-backend/src/ingestion/processors/GithubDiscoveryProcessor.ts

```diff
@@ -78,7 +78,9 @@ export class GithubDiscoveryProcessor implements CatalogProcessor {
     this.logger.info(`Reading GitHub repositories from ${location.target}`);
 
     const { repositories } = await getOrganizationRepositories(client, org);
-    const matching = repositories.filter(r => repoSearchPath.test(r.name));
+    const matching = repositories.filter(
+      r => r.isArchived || repoSearchPath.test(r.name),
```

Comment:
Hmm, I think the logic is backwards here?

```
const matching = repositories.filter(r => !r.isArchived && repoSearchPath.test(r.name));
```

I'm wondering how the test passed, I think you might need to make sure it's only called with the resulting location once:

```
expect(emitter).toHaveBeenCalledTimes(1);
```

Unless I'm missing something?